src/Turbo/README.md

```diff
@@ -166,8 +166,37 @@ class TaskController extends AbstractController
 {
     public function new(Request $request): Response
     {
-        $task = new Task();
+        // Symfony 5.3+
+        return $this->handleForm(
+            $this->createForm(TaskType::class, new Task()),
+            $request,
+            function (FormInterface $form, Task $task) use ($request) {
+                // perform some action with the $task object
+                // such as saving the task to the database...
+                
+                // 🔥 The magic happens here! 🔥
+                if (TurboStreamResponse::STREAM_FORMAT === $request->getPreferredFormat()) {
+                    // If the request comes from Turbo, only send the HTML to update using a TurboStreamResponse
+                    return $this->render(
+                        'task/success.stream.html.twig',
+                        ['task' => $task],
+                        new TurboStreamResponse()
+                    );
+                }
+                
+                // If the client doesn't support JavaScript, or isn't using Turbo, the form still works as usual.
+                // Symfony UX Turbo is all about progressively enhancing your apps!
+                return $this->redirectToRoute('task_success', [], Response::HTTP_SEE_OTHER);
+            },
+            function (FormInterface $form) {
+                return $this->render('task/new.html.twig', [
+                    'form' => $form->createView(),
+                ]);
+            }
+        );
 
+        // Older versions
+        $task = new Task();
```

Comment:
I don't like having the same example twice - we have one giant code block now, with a little "// Older versions" to show that this code block is actually showing the same thing twice.

I'd vote to JUST show the 5.3 format with a comment above saying something like:

```
// See "Form Response Code Changes" above if you are not using the Symfony 5.3 handleForm() method.
```